### ISSUE TITLE
Table > Edit Data > allow multiline edit, improve copy and paste of multiline content (keep line breaks)

### ISSUE THREAD

**Issue by aisbergde** [microsoft/azuredatastudio#14055]
**Is your feature request related to a problem? Please describe.**

Very (very very) often it is required to enter some data directly into mssql (master data, configuration data, ...).
SSMS and ADS can be used to insert or edit 1-line-content. But they can't be used to edit multi-line-content.

- SSMS is not showing the whole content in edit mode, but only the first line. 
- ADS is a bit better:
  - a small part of the content inclusive line breaks is showing in the tooltip
  - but when the content is copied then line breaks are lost
  - it is also not possible to enter multiline data

The only solution I know is to connect other frontends to the database to enter data. For example Access via ODBC.

But I would like to see a way to edit multiline data inside ADS (or inside SSMS) without the need for a separate tool

**Describe the solution or feature you'd like**

enhance multiline edit of cell content:
- allow to resize the cell height (column width can be changed, why not also row height?)
- allow entering and edit multiline-content, there should be a way to enter return. Currently, on "Enter" the focus moves to the next cell. This is OK, you could implement something like Ctrl-Enter, Shift-Enter or Alt-Enter to stay in the line
- additional there should be a context menu item to open the cell content in a separate big sized editor
- Enhance copy and paste from and to lines
  - keep line breaks
- In other words: I want to get some features from Access Forms directly in ADS

An extension adding some form features to edit data would also OK, if it is not possible to extend the existing "edit data" feature.

**Describe alternatives you've considered**
- Access via ODBC
- I am still looking for some other easy to use alternatives, but I can't find for many years
- the issue with Access is: in many companies it is not part of the standard office suite and often a complicated process is required to get it installed
- On the other hand: ADS can be installed without admin right and it would be the "natural place" to enter and edit data without Access

**Comment by aisbergde** [comment]
Maybe as an intermediate step you could implement that the following setting also affects result windows of "Edit Data":

>queryEditor.results.copyRemoveNewLine

At the moment, turning off only works for a "normal" result window.

It would be even better if there were a separate area in the settings under Data next to "queryEditor", for example "dataEditor". Because the Data Editor is different from the normal Query Editor.
